app/javascript/elm/src/Main.elm

```diff
@@ -13,6 +13,7 @@ import Data.Page exposing (Page(..))
 import Data.Path as Path exposing (Path)
 import Data.SelectedSession as SelectedSession exposing (SelectedSession)
 import Data.Session exposing (..)
+import Data.Status exposing (Status(..))
```

Comment:
In general, you could create helpers functions in a Status module in a way that you wouldn't even have to expose the internals of status - I mean the `(..)`
It's more code, but I'm a fan of these small methods. It feels right :)
You can even write very small thingies like `default = Active`
Check next comments and a Theme module if you want to see what I mean.